### PR TITLE
Feedback page

### PR DESCRIPTION
**Pull request by MeenaModhvadia**
- the form_builder function name added to call with properties

### CODE REVIEW

File: web/app/themes/clarity/src/components/c-feedback-form/view.php [hunk 1]
```diff
@@ -6,7 +6,10 @@
 $activeAgency = $oAgency->getCurrentAgency();
 // LEGACY END
 global $wp;
-$current_url = home_url($wp->request)
+$current_url = home_url($wp->request);
+$current_user = wp_get_current_user();
+$display_name = $current_user->display_name;
+$user_email = $current_user->user_email
```
Comment (MalcolmVonMoJ) [inline web/app/themes/clarity/src/components/c-feedback-form/view.php:12]: missing semi-colon?

**Review by MalcolmVonMoJ — APPROVED** [review]
Looks good, but there appears to be a missing semi-colon to line 12.

**Review by brown-a2 — COMMENTED** [review]
Make sure to add the .gitkeep file back in

**Comment by brown-a2** [comment]
Also, just curious, I'm probably going blind, but where are the two variables $current_user and being $current_url used?

**Comment by MalcolmVonMoJ** [comment]
$current_user is being used in the 2 lines beneath where it is being declared.  
$current_url was there previously, just that the missing semi colon was added in.

**Comment by MeenaModhvadia** [comment]
How do I make sure to add the .gitkeep file back in?

**Comment by wilson1000-MoJ** [comment]
Hi Meena,
I've popped some info on the PR request for you:

https://mojdt.slack.com/archives/C03QE40GVA6/p1658833449507179?thread_ts=1658830348.942229&cid=C03QE40GVA6

I hope that is helpful :)

**Pull request merged**